disorder-core/disorder-core.cabal

```diff
@@ -17,6 +17,7 @@ library
                      , text                            >= 1.1        && < 1.3
                      , process                         == 1.2.*
                      , QuickCheck                      == 2.7.*
+                     , ieee754                         == 0.7.*
```

Comment:
Just one possible issue with this, is it a stable version, likely to be compatible with many other packages we are using? Or can it be a cause of cabal hell?